### DISCUSSION TITLE
config/Independent

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#221] [Announcements]
# config/Independent

介绍如何在存量VitePress中引入使用或者使用主题中的部分功能

http://localhost:5173/config/Independent.html

**Comment by HyjoyX** [comment]
大佬，咱这个后面有打算优化关于构建相关的计划不，在自己的项目里引用了主题，然后里面有根据nav自动生成目录相关的但是每次更新nav，构建总是崩掉，总是需要重启，除此之外继承了BlogTheme，Layout自定义好像就不是很方便了，以及配置的vitepress-plugin-nprogress进度条好像也不兼容了，偶尔出现bug，进度条始终加载不完

**Reply by ATQQ** [reply]
你把你问题仓库留一下

我后续看着问题优化一下，还有你电脑配置发一下

**Reply by HyjoyX** [reply]
okok，仓库地址：https://github.com/AutumnFishs/blog
node：18.20.2
那个电脑配置啥

**Reply by ATQQ** [reply]
不是构建崩溃嘛，看看你电脑内存多少G的 hh
